client/fcm.js

```diff
@@ -0,0 +1,41 @@
+export async function initializeFCM(messageSenderId) {
+  if ( !messageSenderId ){
+      console.log("Cannot subscribe to notifications at this moment");
+      return;
```

Comment:
can we return false and change the message to message sender id is required or something